### PR TITLE
Update diagnostics bundle docs

### PR DESCRIPTION
**Pull request by kilfoyle**
Diagnostic bundles now have a `delete` button, so this updates the screen capture and makes a note of the delete option. Sections affected are:
 - [Collect Elastic Agent diagnostics](https://www.elastic.co/guide/en/fleet/8.13/monitor-elastic-agent.html#collect-agent-diagnostics) (on the Agent monitoring page)
 - [Get the diagnostics bundle through Fleet](https://www.elastic.co/guide/en/fleet/8.13/fleet-troubleshooting.html#trb-collect-agent-diagnostics) (on the Troubleshooting page)

Target: 8.15
Closes: #1080

---
PREVIEW

Screenshot 2024-05-17 at 9 13 57 AM

### CODE REVIEW

File: docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc [hunk 1]
```diff
@@ -178,7 +178,7 @@ image::images/collect-agent-diagnostics.png[Collect agent diagnostics under agen
 
 Any in-progress or previously collected bundles for the {agent} will be listed on this page.
 
-Note that the bundles are stored in {es} and will be removed after 30 days.
+Note that the bundles are stored in {es} and will be removed after 30 days. You can also delete any previously created bundle by clicking the `trash can` icon.
```
Comment (jen-huang) [inline docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc:181]: actually the [default ILM policy](https://github.com/elastic/elasticsearch/blob/5a52642db705b450ab14efa86c781462516711d5/x-pack/plugin/core/template-resources/src/main/resources/fleet-file-fromhost-data-ilm-policy.json#L13) for where these diagnostics files are stored deletes them after 7 days, this wasn't changed recently so not sure where the original info came from 😅

File: docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc [hunk 2]
```diff
@@ -427,11 +427,11 @@ To download the diagnostics bundle for local viewing:
 . Click the **Request diagnostics .zip** button.
 +
 [role="screenshot"]
-image::images/collect-agent-diagnostics2.png[Collect agent diagnostics under agent details]
+image::images/collect-agent-diagnostics.png[Collect agent diagnostics under agent details]
 
 Any in-progress or previously collected bundles for the {agent} will be listed on this page.
 
-Note that the bundles are stored in {es} and will be removed after 30 days.
+Note that the bundles are stored in {es} and will be removed after 30 days. You can also delete any previously created bundle by clicking the `trash can` icon.
```
Comment (jen-huang) [inline docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:434]: same correction needed here

File: docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc (hunk 1 above)
Comment (kilfoyle) [reply docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc]: Thanks! Fixed in [2911cb9](https://github.com/elastic/ingest-docs/pull/1081/commits/2911cb925942d0b9df02138ac5cd7455c65857d2).

File: docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc (hunk 2 above)
Comment (kilfoyle) [reply docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc]: Thanks! Fixed in [2911cb9](https://github.com/elastic/ingest-docs/pull/1081/commits/2911cb925942d0b9df02138ac5cd7455c65857d2).

**Review by karenzone — APPROVED** [review]
LGTM!

**Pull request merged**